packages/docs/src/mixins/codepen.js

```diff
@@ -19,13 +21,20 @@ export default {
     },
     boot (res) {
       const template = this.parseTemplate('template', res)
+      let templatePug = template
+      if (templatePug) {
+        templatePug = templatePug.slice('<template>'.length).slice(0, -'</template>'.length)
+        templatePug = convert(templatePug, { bodyLess: true, attrComma: true, doubleQuotes: true, encode: false })
+        templatePug = `<template lang="pug">\n${templatePug}</template>`
+      }
       const style = this.parseTemplate('style', res)
       const script = this.parseTemplate('script', res)
       const codepenResources = this.parseTemplate('codepen-resources', res)
       const codepenAdditional = this.parseTemplate('codepen-additional', res)
 
       this.pen = {
         template,
+        'template-pug': templatePug,
```

Comment:
I also thought about a global button to just always use `pug` templates like light/dark mode setting :thinking:  
But this would be much harder to implement